controllers/selfnoderemediation_controller.go

```diff
@@ -483,6 +485,7 @@ func (r *SelfNodeRemediationReconciler) recoverNode(node *v1.Node, snr *v1alpha1
 			return ctrl.Result{}, err
 		}
 		r.Recorder.Event(snr, eventTypeNormal, eventReasonRemoveFinalizer, "Remediation process - remove finalizer from snr")
```

Comment:
Shouldn't this use `events.NormalEvent` from `common`?

Reply:
IIRC the common events PR have been added after SNR events PR was merged. I guess we didn't update SNR with the common events API

Reply:
I was asking becase in the line below we use `events.RemediationFinished`, which should come from the common events API

Reply:
Good point, TBH I didn't think of that .
I've only considered the "special" events.
I'll update in another PR

Reply:
Here: https://github.com/medik8s/self-node-remediation/pull/178